pyproject.toml

```diff
@@ -75,73 +71,73 @@ module = ["pytket.*"]
 implicit_reexport = true
 
 [[tool.mypy.overrides]]
-# `mqt.bench` can be removed once it is typed.
-# `sb3_contrib` is fully typed but takes forever to install in the environment.
+# `mqt.bench` and `sb3_contrib` are fully typed but take forever to install in the environment.
 # recent versions of `gym` are typed, but stable-baselines3 pins a very old version of gym.
 # qiskit is not yet marked as typed, but is typed mostly.
 # the other libraries do not have type stubs.
-module = ["qiskit.*", "joblib.*", "sklearn.*", "matplotlib.*", "mqt.bench.*", "gymnasium.*", "sb3_contrib.*"]
+module = ["qiskit.*", "joblib.*", "sklearn.*", "matplotlib.*", "gymnasium.*", "mqt.bench.*", "sb3_contrib.*"]
 ignore_missing_imports = true
 
 [tool.ruff]
-select = [
-    "E", "F", "W", # flake8
+line-length = 120
+src = ["src"]
+unsafe-fixes = true
```

Comment:
I believe you are missing one of the import settings from our other projects here: the one that adds jupyter notebooks to the files that should be checked. Would you mind checking that in the other repos as well?

Reply:
Added it. Will also do for the other repos.